src/routes/webhooks/paddle.ts

```diff
@@ -88,11 +89,36 @@ const updateUserSubscription = async ({
   );
 };
 
+const getUserId = async ({
+  subscriptionId,
+  userId,
+}: {
+  subscriptionId?: false | string | null;
+  userId?: string | undefined;
+}): Promise<string> => {
+  if (userId) {
+    return userId;
+  }
+
+  const con = await createOrGetConnection();
+  const user = await con.getRepository(User).findOne({
+    where: { subscriptionFlags: JsonContains({ subscriptionId }) },
```

Comment:
we should add an index

Reply:
Added gin index because we query on top level here.
Tested on prod and it catches it.

![Screenshot 2024-11-18 at 09 15 58](https://github.com/user-attachments/assets/27ef6ab6-ec5a-4545-8b8b-e76128132686)